net-analyzer/wireshark/wireshark-9999.ebuild

```diff
@@ -147,6 +154,21 @@ pkg_setup() {
 	python-any-r1_pkg_setup
 }
 
+src_unpack() {
+	if [[ ${PV} == *9999* ]] ; then
+		git-r3_src_unpack
+	else
+		if use verify-sig ; then
+			verify-sig_verify_signed_checksums \
+				"${DISTDIR}"/${P}-signatures.txt \
+				sha256 \
+				"${DISTDIR}"/${P}.tar.xz
```

Comment:
No cd / openssl-dsgt in this one? Not that it matters in live (yet), but just wondering if an oversight.

Beside that don't really have anything to say from a quick look. I'd probably have considered BASH_REMATCH in the eclass but utimately doesn't change much.

Reply:
Yes, oversight. I've only modified the one ebuild I needed to test the eclass.